infrastructure/prod.tfvars

```diff
@@ -1,7 +1,13 @@
 vault_section = "prod"
 capacity = "2"
+reupload_delay = "10000"
+reupload_enabled = true
+scan_delay = "10000"
+scan_enabled = true
```

Comment:
booleans have to be wrapped in double quotes

Reply:
done